src/handlers/handlers.js

```diff
@@ -0,0 +1,17 @@
+var fs = require('fs');
+var path = require('path');
+
+var handlers = (module.exports = {});
+
+handlers.serveLanding = function(request, response) {
+  fs.readFile(
+    path.join(__dirname, '..', '..', 'public', 'index.html'),
+    function(err, file) {
+      if (err) {
+        throw err;
```

Comment:
Would prefer not to plant the idea of throwing in a handler. I've been commenting during code reviews about this during every cohort. I can see you've used the proper approach in other steps, so perhaps this was an oversight?